release/azure-pipelines-release.yml

```diff
@@ -73,7 +73,7 @@ jobs:
       # --jobs=1 is necessary due to resource constraints on the build machine.
       # Running multiple Vivado and Verilator build jobs simultaneously can
       # cause Synthesis failures
-      ci/bazelisk.sh build release --jobs=1
+      ci/bazelisk.sh build release --jobs=1 --define bitstream=vivado
 
       # Set the package-path variable for the artifact-publishing step
       PKG_PATH=$(./bazelisk.sh outquery //release:opentitan)
```

Comment:
It's possible this line will need it too. I feel like I've seen cquery pull in external repos.

Reply:
Good call -- I've added it just in case.